test-specifications/crypto-service/accountUpdateTransaction.md

```diff
@@ -19,111 +17,440 @@ https://github.com/hashgraph/hedera-protobufs/blob/main/services/crypto_update.p
+|---------|------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Updates an account with an auto renew period set to 60 days (5,184,000 seconds)                                        | accountId=<CREATED_ACCOUNT_ID>, autoRenewPeriod=5184000, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update succeeds and the account's auto renew period should equal 5,184,000 seconds.  | N                 |
+| 2       | Updates an account with an auto renew period set to -1 seconds                                                         | accountId=<CREATED_ACCOUNT_ID>, autoRenewPeriod=-1, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>]      | The account update fails with an INVALID_RENEWAL_PERIOD response code from the network.          | N                 |
+| 3       | Updates an account with an auto renew period set to the minimum period of 30 days (2,592,000 seconds)                  | accountId=<CREATED_ACCOUNT_ID>, autoRenewPeriod=2592000, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update succeeds and the account's auto renew period should equal 2,592,000 seconds.  | N                 |
+| 4       | Updates an account with an auto renew period set to the minimum period of 30 days minus one second (2,591,999 seconds) | accountId=<CREATED_ACCOUNT_ID>, autoRenewPeriod=2591999, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update fails with an AUTORENEW_DURATION_NOT_IN_RANGE response code from the network. | N                 |
+| 5       | Updates an account with an auto renew period set to the maximum period of 8,000,001 seconds                            | accountId=<CREATED_ACCOUNT_ID>, autoRenewPeriod=8000001, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update succeeds and the account's auto renew period should equal 8,000,001 seconds.  | N                 |
+| 6       | Updates an account with an auto renew period set to the maximum period plus one seconds (8,000,002 seconds)            | accountId=<CREATED_ACCOUNT_ID>, autoRenewPeriod=8000002, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update fails with an AUTORENEW_DURATION_NOT_IN_RANGE response code from the network. | N                 |
```

Comment:
Should be 'plus one second' (also there are similar places like this one).

Reply:
I guess I missed some edits from when I copy-pasted from `accountCreateTransaction.md`.. fixed